contracts/MTF.sol

```diff
@@ -85,8 +85,8 @@ contract MTF is MintableToken, Ownable {
         // Receiver3 2.0% 0x733bc7201261aC3c9508D20a811D99179304240a
         mint(0x733bc7201261aC3c9508D20a811D99179304240a, toDistribute.mul(2).div(100));
 
-        // Receiver4 8.0% 0x4b6716bd349dC65d07152844ed4990C2077cF1a7
-        mint(0x4b6716bd349dC65d07152844ed4990C2077cF1a7, toDistribute.mul(8).div(100));
+        // Receiver4 18.0% 0x4b6716bd349dC65d07152844ed4990C2077cF1a7
```

Comment:
This change is not sufficient, as ICO token distribution part should be decreased from 60% to 50%. It's required to update line 70 `toDistribute` variable value to reflect this change.

Reply:
And unit tests must be updated accordingly, they will fail after recent changes.